billing/details.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Update the contact information
+description: Information on how update contact information in Docker Hub
+keywords: payments, billing, subscription, contact
+---
+
+Find information on how to update the contact information for your personal account or for an organization. 
```

Comment:
**question:** I think we need to rename `contact information`  → `billing information`. 

What do you think? If you agree, I can make UI changes 👇🏻 

<img width="1023" alt="Screenshot 2023-03-03 at 2 26 11 PM" src="https://user-images.githubusercontent.com/2487951/222842500-f1a0f659-b639-4fc4-ae79-3a59aae3cf36.png">


```suggestion
Find information on how to update the billing information for your personal account or for an organization. 
```

Reply:
Yes I definitely agree. But would you mind holding off on making that change for now? Whilst I was writing the docs I noticed a few UI copy inconsistencies within the billing section of Hub (nothing major, just some things not aligning with our Style Guide). I planned on gathering all the changes into one ticket, so the UI can be updated all in one go. Would that work for you?

Reply:
Okay, let me start a PR to rename it.

> Would it be possible to also change all/most of the copy to sentence case?

Could you please start a discussion in `#hub-frontend` channel on Slack?